### PR TITLE
Add api for switch to real time and warning update

### PR DESCRIPTION
**Pull request by arthurkkhi**

### CODE REVIEW

File: Source/Lib/Codec/EbEncHandle.c [hunk 1]
```diff
@@ -552,10 +552,31 @@ EB_U64 GetAffinityMask(EB_U32 lpnum) {
 }
 #endif
 
+void SwitchToRealTime()
+{
+
+#if  __linux__
+
+    struct sched_param schedParam = {
+        .sched_priority = 99
```
Comment (tianjunwork) [inline Source/Lib/Codec/EbEncHandle.c]: use sched_get_priority_max(SCHED_FIFO) instead of 99?

Comment (hassount) [reply Source/Lib/Codec/EbEncHandle.c]: That would freeze the machine :)

File: Source/App/EbAppConfig.c [hunk 2]
```diff
@@ -95,6 +95,7 @@
 #define ASM_TYPE_TOKEN				    "-asm" // no Eval
 #define THREAD_MGMNT                    "-lp"
 #define TARGET_SOCKET                   "-ss"
+#define SWITCHTHREADSTOREALTIME_TOKEN   "-switchThreadsToRtPriority"
```
Comment (tianjunwork) [inline Source/App/EbAppConfig.c]: Could we use a shorter name for the api? internally it is fine to use a more readable name.

File: Config/Sample.cfg [hunk 3]
```diff
@@ -117,6 +117,7 @@ RegisteredUserData              : 0             # SEI message (0= OFF, 1=ON )
 UnregisteredUserData            : 0             # SEI message (0= OFF, 1=ON )
 RecoveryPoint                   : 0             # SEI message (0= OFF, 1=ON )
 TemporalId                      : 1             # 0: OFF, 1: Insert temporal ID in NAL units 
+SwitchThreadsToRtPriority       : 0             # 0: OFF, 1: Switch threads to real time priority
```
Comment (tianjunwork) [inline Config/Sample.cfg]: This change will make a great latency difference. Do you want to keep "SwitchThreadsToRtPriority" default to 1?

**Comment by tianjunwork** [comment]
please update the user guide also.

File: Source/Lib/Codec/EbEncHandle.c (hunk 1 above)
Comment (tianjunwork) [reply Source/Lib/Codec/EbEncHandle.c]: I thought sched_get_priority_max(SCHED_FIFO) is 99?

File: Source/App/EbAppConfig.c (hunk 2 above)
Comment (arthurkkhi) [reply Source/App/EbAppConfig.c]: How about "-real-time" or "-real-time-thread"?

Comment (tianjunwork) [reply Source/App/EbAppConfig.c]: much better, or "-rt"?

File: Config/Sample.cfg (hunk 3 above)
Comment (dseah-intel) [reply Config/Sample.cfg]: Default has be set to 1

File: Source/App/EbAppConfig.c (hunk 2 above)
Comment (arthurkkhi) [reply Source/App/EbAppConfig.c]: set to "-rt"

File: Source/Lib/Codec/EbEncHandle.c (hunk 1 above)
Comment (arthurkkhi) [reply Source/Lib/Codec/EbEncHandle.c]: set to sched_get_priority_max(SCHED_FIFO)

**Comment by dseah-intel** [comment]
@tianjunwork User guide has been updated :)

**Pull request merged**